pkg/cmd/server/kubernetes/node/node_config.go

```diff
@@ -34,6 +31,9 @@ type NodeConfig struct {
 	AllowDisabledDocker bool
 	// Containerized is true if we are expected to be running inside of a container
 	Containerized bool
+	// DNSClient is a client that is only used to lookup default DNS IP addresses on
+	// the cluster. It should not be passed into the Kubelet.
+	DNSClient kclientsetexternal.Interface
```

Comment:
Why not build this locally in the `EnsureLocalQuota` to make sure it doesn't escape?

Reply:
Doesn't have access to masterconfig.

Reply:
> Doesn't have access to masterconfig.

nodeConfig.MasterKubeConfig right?